### PR TITLE
If feature=force_aesni_support then disable the AES T-table code

### PR DESCRIPTION
**Pull request by jack-fortanix**
This prevents any possibility of calling side-channel insecure code, and also reduces code size.

### CODE REVIEW

File: mbedtls-sys/Cargo.toml [hunk 1]
```diff
@@ -36,6 +36,7 @@ default = ["std", "zlib", "time", "pthread", "aesni", "padlock", "legacy_protoco
 std = []
 custom_printf = []
 custom_has_support = []
+force_aesni_support = []
```
Comment (jethrogb) [inline mbedtls-sys/Cargo.toml]: The feature in this create should just be called `aes_alt`

**Comment by jethrogb** [comment]
Do we test force_aesni_support in CI right now? If so r=me. Otherwise, please add that and then r=me.

**Comment by jack-fortanix** [comment]
Build was tested (via sgx target) but was not running the tests. Added.

**Pull request merged**